src/components/Header.jsx

```diff
@@ -4,9 +4,20 @@ import ProfileIconDump from '../assets/images/ProfileRed.svg';
 import NavigationBar from './NavigationBar';
 import Divider from './Divider';
 import PageTitle from './PageTitle';
-import Footer from './Footer';
 
-const Header = () => {
+// Header
+/**
+ * @param {boolean} [isNavigationBar=false] isNavigationBar Navigation 활성화 유무
+ * @param {boolean} [isPageTitle=false] isPageTitle PageTitle 활성화 유무
+ * @param {string | undefined} viewName PageTitle에 들어갈 viewName 값
+ * @param {string} [navigationViewName=PreviousView] navigationViewName Navigation에 들어갈 viewName 값
+ */
+const Header = ({
+    isNavigationBar = false,
+    isPageTitle = false,
+    viewName = undefined,
+    navigationViewName = 'PreviousView',
+}) => {
```

Comment:
viewName, navigationViewName 이 비어있을때는 컴포넌트 비활성화, 비어있지 않은경우(값이 들어올 경우) 활성화 시키면 좀 더 간결해지지 않을까요?
```suggestion
 * @param {string | undefined} viewName PageTitle에 들어갈 viewName 값
 * @param {string | undefined} [navigationViewName=PreviousView] navigationViewName Navigation에 들어갈 viewName 값
 */
const Header = ({
    viewName = undefined,
    navigationViewName = undefined,
}) => {
```

이와 같이 적용하면 컴포넌트 사용이 좀 더 쉬워질 것 같습니다